train.sh

```diff
@@ -6,7 +6,7 @@
 
 parse_args "$0" "experiment dataset model load_from" "$@"
 shift $n
-check_config "IAM_ROLE OWNER DATASET_OWNER IMAGE TRAIN_TASK_NAME"
+check_config "IAM_ROLE OWNER DATASET_OWNER IMAGE PROJECT TRAIN_TASK_NAME"
 
 load_from=$(check_aws $load_from)
```

Comment:
can't we have load_from defaults to None?

Reply:
yeah. feel free to do that in this PR.

Reply:
Separate PR for that because it needs changes to lib.sh to parse arguments with defaults.